hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -108,14 +108,17 @@
   private transient Timer.Context indexTimer = null;
 
   /**
-   *
+   * @param jsc Java Spark Context
```

Comment:
if possible add a line about instantiation and what this constructor is used for compared to other constructors.